diffrax/_solver/srk.py

```diff
@@ -281,16 +282,24 @@ def __post_init__(self):
     Should be a NumPy array of shape `(s-1,)`, as the first stage has time increment 0.
 - `cfs_bm`: An instance of a subclass of `AbstractStochasticTableau` representing
     the coefficients for the Brownian increment and possibly its Levy areas.
+- `ignore_stage_f`: Optional. A NumPy array of length `s` of booleans. If `True` at
+    stage `j`, the vector field of the drift term will not be evaluated at stage `j`.
+- `ignore_stage_g`: Optional. A NumPy array of length `s` of booleans. If `True` at
+    stage `j`, the diffusion vector field will not be evaluated at stage `j`.
 """
 
 
 class AbstractSRK(AbstractSolver[_SolverState]):
     r"""A general Stochastic Runge-Kutta method.
 
-    The second term in the MultiTerm must be a `ControlTerm` with
-    `control=VirtualBrownianTree`. Depending on the Butcher tableau, the
-    `VirtualBrownianTree` may need to be initialised with 'levy_area="space-time"'
-    or 'levy_area="space-time-time"'.
+    This accepts `terms` of the form
+    `MultiTerm(ODETerm(drift), ControlTerm(diffusion, brownian_motion))`.
+     Depending on the solver, the brownian motion might need to generate
+    different types of Levy areas, specified by the `minimal_levy_area` attribute.
+
+    For example, the ShARK solver requires space-time Levy area, so it will have
```

Comment:
You can cross-link (in the docs) by writing this as ``[`diffrax.ShARK`][]``.

Reply:
Oh nice, that's a good trick to know.